scene/gui/code_edit.h

```diff
@@ -465,6 +465,8 @@ class CodeEdit : public TextEdit {
 
 	void set_symbol_lookup_word_as_valid(bool p_valid);
 
+	void duplicate_lines();
```

Comment:
`nit:` Can title this with a new section i.e from `TextEdit`:  `// Text manipulation` or `// Input actions`